src/msg/liquid/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Coin } from '../../types/Coin';
+
+function newMsgMintDerivative(
+  sender: string,
+  validator: string,
+  amount: Coin[]
```

Comment:
Confirmed with Ruaridh that `amount` in both functions should be a singular `Coin`.